ajax/libs/oimo/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "oimo",
+  "version": "1.0.9",
+  "description": "JavaScript 3D physics engine.",
+  "filename": "oimo.min.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/lo-th/oimo.git"
+  },
+  "keywords": [
+    "oimo",
+    "oimo.js",
+    "physics",
+    "3d"
+  ],
+  "author": "lo-th",
+  "license": "MIT",
+  "homepage": "https://lo-th.github.io/Oimo.js/",
+  "npmName": "oimo",
+  "npmFileMap": [
+    {
+      "basePath": "build",
+      "files": [
+        "oimo.!(module.js)"
```

Comment:
`**/!(oimo.module.js)` might be a better choice because the files we need are located in `build`?